packages/qwik-city/src/buildtime/vite/dev-server.ts

```diff
@@ -227,10 +228,27 @@ export function ssrDevMiddleware(ctx: BuildContext, server: ViteDevServer) {
             await server.ssrLoadModule('@qwik-serializer');
           const qwikSerializer = { _deserializeData, _serializeData, _verifySerializable };
 
+          const applyRewrite: ApplyRewriteInternal = async (url: URL) => {
```

Comment:
TODO - not sure this is the best name to this function.
it doesnt actually apply the rewrite itself.